src/views/Profile/AuthorsList/AuthorsTableBody.tsx

```diff
@@ -32,12 +50,27 @@ const AuthorsTableBody: React.FC<IAuthorsTableBodyProps> = ({ authors }) => {
             {fullName}
           </Typography>
         </TableCell>
-        <TableCell>{region}</TableCell>
-        <TableCell>City</TableCell>
-        <TableCell>{dateOfCreation}</TableCell>
+        <TableCell>{region?.name}</TableCell>
+        <TableCell>{mainInstitution?.city.name}</TableCell>
+        <TableCell>{parsedCreatedAt}</TableCell>
 
-        <TableCell>{dateOfEdition}</TableCell>
-        <TableCell>Action</TableCell>
+        <TableCell>{parsedEditedAt}</TableCell>
+        <TableCell>
+          <div className={classes.modifSection}>
+            <IconButton
+              aria-label="edit profile"
+              onClick={() => handleChangeClick(id)}
+              className={classes.editButton}
+            >
+              <EditOutlined />
+            </IconButton>
+            <AuthorsDeletingDialog
```

Comment:
I think it will be much more flexible, readable and better for testing if we put conditional rendering here:

```
{isAllowedToDelete ? 
  <AuthorsDeletingDialog id={id} fullName={fullName} /> :
  <AuthorCannotBeDeletedTooltip fullName={fullName} />
}
```